llvm/lib/Transforms/Scalar/IR2Vec-LOF/LoopDistributionServer/LoopDistributionServer.cpp

```diff
@@ -118,15 +115,12 @@ struct LoopDistributionServerPass
       const ::loopdistribution::LoopDistributionRequest *request,
       ::loopdistribution::LoopDistributionResponse *response) override {
 
-    errs() << "came here\n";
     std::string partition = request->partitionpattern();
 
     if (partition == "Exit") {
-      errs() << "server exit requested\n";
-      AOTRunner->requestExit();
+      // AOTRunner->requestExit();  HAVE TO UNCOMMENT LATER
```

Comment:
Is this line not needed?

Reply:
gRPC flow is not fixed. Hence we can take this later.